OpenCDMi/FrameworkRPC.cpp

```diff
@@ -1233,6 +1237,118 @@ namespace Plugin {
             Core::JSON::ArrayType<Systems> KeySystems;
         };
 
+        class WidevineConfig : public Core::JSON::Container {
```

Comment:
If possible try to move WidevineConfig and ASConfig into separate file. It is our custom change that probably would never be upstreamed ... so to make our future merges easier ...  would be nice to have that in separate file. Probably it also reduce number of includes in top of that file.